src/Form/Form.js

```diff
@@ -264,6 +264,7 @@ class Form {
     }
 
     removeInputDecoration (input) {
+        removeInlineStyles(input, getBasicStyles(input, ''))
```

Comment:
Not super happy about this, but I couldn't find a better solution. My concern has to do with what I mentioned above. We use this `removeInlineStyles` and pass in the styles we know we've added so we avoid removing stuff we didn't add. Now, when with this change, we're basically saying remove all the styles we _may_ have added, even though we haven't. This introduces a tiny tiny chance for breakage where we'd remove styles we didn't add. The risk is small enough because the styles are very specific.

We need to do this because at this stage in-context is already switched off, so the two lines below won't work.

Reply:
One thing we may do is to introduce a further conditional in the `getIdentitiesIcon` that if the input already has the Dax icon we should return it. I can try it and see how ugly it is. My hunch is "horrible" 🙂.

Reply:
Or if we could remove the styles before changing the in-context state, so that we know what to remove.